tests/conftest.py

```diff
@@ -4,24 +4,27 @@
 
 
 @pytest.fixture
-def pyramid_settings():
-    return {
-        "client_embed_url": "http://hypothes.is/embed.js",
-        "nginx_server": "http://via.hypothes.is",
-        "via_html_url": "https://viahtml.hypothes.is/proxy",
-        "checkmate_url": "http://localhost:9099",
-        "nginx_secure_link_secret": "not_a_secret",
-        "via_secret": "not_a_secret",
-        "signed_urls_required": False,
-        "checkmate_api_key": "dev_api_key",
-        "checkmate_ignore_reasons": None,
-        "checkmate_allow_all": False,
-        "enable_front_page": True,
-        "data_directory": "tests/data_directory",
-        "dev": False,
-        "youtube_transcripts": True,
-        "api_jwt_secret": "secret",
-    }
+def pyramid_settings(pyramid_settings):
+    pyramid_settings.update(
+        {
+            "client_embed_url": "http://hypothes.is/embed.js",
+            "nginx_server": "http://via.hypothes.is",
+            "via_html_url": "https://viahtml.hypothes.is/proxy",
+            "checkmate_url": "http://localhost:9099",
+            "nginx_secure_link_secret": "not_a_secret",
+            "via_secret": "not_a_secret",
+            "signed_urls_required": False,
+            "checkmate_api_key": "dev_api_key",
+            "checkmate_ignore_reasons": None,
+            "checkmate_allow_all": False,
+            "enable_front_page": True,
+            "data_directory": "tests/data_directory",
+            "dev": False,
+            "youtube_transcripts": True,
+            "api_jwt_secret": "secret",
+        }
+    )
+    return pyramid_settings
```

Comment:
Via inherits the cookiecutter's `pyramid_settingsa fixture (which already has the `database_url` setting in it) and then extends it with custom settings